### PR TITLE
Constructed characteristics object for post request; fine tuned to meet API requirements; successfully implemented post request; minor code cleanup and style changes

### PR DESCRIPTION
**Pull request by moneycabui**

### CODE REVIEW

File: client/src/components/ReviewAddFormModal/index.jsx [hunk 1]
```diff
@@ -13,19 +12,19 @@ const ReviewAddFormModal = (props) => {
...
+        <GrClose className={styles.GrClose} onClick={closeAddReviewModal} />
+        <ReviewsAddForm
+          className={styles.modalContent}
+          addReview={addReview}
+          closeModalButton={closeModalButton}
+          closeAddReviewModal={closeAddReviewModal}
+          characteristicNames={characteristicNames}
+          characteristicIds={characteristicIds}
+        />
+      </div>
```
Comment (moneycabui) [inline client/src/components/ReviewAddFormModal/index.jsx:26]: No changes here - just moving stuff around

File: client/src/components/ReviewRatingDistribution/index.jsx [hunk 2]
```diff
@@ -20,27 +20,42 @@ const ReviewRatingDistribution = (props) => {
       <div className={styles.distributionWrapper}>
         <label htmlFor="fiveStar">5 Stars: </label>
         <progress id="fiveStar" max="100" value={fiveStarDistribution || ''}>{`${fiveStarDistribution}%`}</progress>
-        <span> {fiveStarTotal}</span>
+        <span>
+          {' '}
+          {fiveStarTotal}
+        </span>
```
Comment (moneycabui) [inline client/src/components/ReviewRatingDistribution/index.jsx:26]: Fixing ESLint errors

File: client/src/components/ReviewTileStars/index.jsx [hunk 3]
```diff
@@ -7,8 +6,8 @@ const ReviewTileStars = (props) => {
   const remainingStars = 5 - ratings;
   return (
     <div>
-      {[...Array(ratings)].map((star, index) => <FaStar size={20} color="#F5618E" key={`Solid stars ${index}`} />)}
-      {[...Array(remainingStars)].map((star, index) => <FaRegStar size={20} color="#F5618E" key={`Unfilled stars ${index}`} />)}
+      {[...Array(ratings)].map(() => <FaStar size={20} color="#F5618E" key={`SolidStars${(Math.random() * 20)}`} />)}
+      {[...Array(remainingStars)].map(() => <FaRegStar size={20} color="#F5618E" key={`UnfilledStars${(Math.random() * 20)}`} />)}
```
Comment (moneycabui) [inline client/src/components/ReviewTileStars/index.jsx:10]: Fixing console key warnings

File: client/src/components/ReviewsList/index.jsx [hunk 4]
```diff
@@ -72,55 +68,10 @@ class ReviewsList extends React.Component {
...
-      photos: formData.uploadedFile || '',
-      characteristics: {
-        sizeId: formData.size,
-        widthId: formData.width,
-        comfortId: formData.comfort,
-        qualityId: formData.quality,
-        productLengthId: formData.productLength,
-        fitId: formData.fit,
-      },
-    };
```
Comment (moneycabui) [inline client/src/components/ReviewsList/index.jsx:99]: Most of these did not get deleted, but were moved up and down (ESLint)

File: client/src/components/ReviewsList/index.jsx [hunk 5]
```diff
@@ -152,11 +103,79 @@ class ReviewsList extends React.Component {
...
+
+    axios.post('/reviews', reviewDataObject)
+      .then((response) => {
+        console.log('Successfully added review: ', response.data);
+        this.getReviews();
+      })
+      .catch((error) => {
+        console.log('Error adding review: ', error);
+      });
+  }
```
Comment (moneycabui) [inline client/src/components/ReviewsList/index.jsx:167]: This is the bulk of the work done in this PR

**Review by agentanky — APPROVED** [review]

**Pull request merged**